c7n/resources/rdscluster.py

```diff
@@ -40,7 +40,7 @@ class RDSCluster(QueryResourceManager):
     class resource_type(object):
 
         service = 'rds'
-        type = 'rds-cluster'
+        type = 'cluster'
```

Comment:
@kapilt - I changed this because otherwise the arn is wrong.  But if changing this has implications on other areas I can override `generate_arn` to use the correct type.  (I searched for `type` elsewhere and I don't think it does, but it is a very overloaded term in the code base)